### PR TITLE
feat: implement missed methods

### PR DESCRIPTION
**Pull request by Alex-A4**
## Status

**READY**

## Description

- Added `hash` field for return type for methods `codeToTvc`, `getExpectedAddress`, `mergeTvc`, `setCodeSalt`, `packIntoCell`
- Implemented missed methods for transport: `getBlockchainConfig`, `getContractFields`, `getDstTransaction`
- Implemented missed abi methods: `executeLocal`, `unpackInitData`, `unpackContractFilds`

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/flutter_nekoton_bridge/example/ios/Podfile.lock [hunk 1]
```diff
@@ -24,4 +24,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ef19549a9bc3046e7bb7d2fab4d021637c0c58a3
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.12.0
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/example/ios/Podfile.lock:27]: whaaat? :(

File: packages/flutter_nekoton_bridge/lib/nekoton/helpers/abi.dart [hunk 2]
```diff
@@ -270,35 +270,117 @@ Future<PublicKey> extractPublicKey(String boc) async {
...
+  if (result.length == 1) {
+    throw ExecuteLocalException(result[0]);
+  }
+
+  return (result[0], Transaction.fromJson(jsonDecode(result[1])));
+}
+
+/// Unpack data from [contractAbi].
+///Returns optional public key and json-encoded Map<String, Token>
+///or throws error.
```
Comment (nesquikm) [inline packages/flutter_nekoton_bridge/lib/nekoton/helpers/abi.dart:355]: Just a formatting: absent space between /// and text.

**Review by nesquikm — APPROVED** [review]

File: packages/flutter_nekoton_bridge/example/ios/Podfile.lock (hunk 1 above)
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/example/ios/Podfile.lock:27]: -

File: packages/flutter_nekoton_bridge/lib/nekoton/helpers/abi.dart (hunk 2 above)
Comment (Alex-A4) [reply packages/flutter_nekoton_bridge/lib/nekoton/helpers/abi.dart:355]: -

**Pull request merged**